src/types.ts

```diff
@@ -0,0 +1,8 @@
+export type TTile = {
+    width: string;
+    height: string;
+    content: string;
+    textSize?: string;
+    imgOptions?: string[];
```

Comment:
Don't see why it has to be an array when other class props are passed as a list

Reply:
The reason is it isn't necessary explicitly define all possible options for img style classes but classes are added via node.classList.add() method which cannot parse several classes provided as one string, only as separated parameters